manifests/init.pp

```diff
@@ -3,38 +3,38 @@
-  $check_changed_options             = 'verbose',
-  $check_changed_deps                = 'yes',
-  String $pkg_repo_signing_key       = '',
-  $parallel_jobs                     = $facts['processors']['count'],
-  String $save_wrkdir                = '',
-  String $wrkdir_archive_format      = '',
-  String $nolinux                    = '',
-  String $no_package_building        = '',
-  String $no_restricted              = '',
-  String $allow_make_jobs            = '',
-  String $url_base                   = '',
-  String $max_execution_time         = '',
-  String $nohang_time                = '',
-  String $http_proxy                 = '',
-  String $ftp_proxy                  = '',
-  String $build_as_non_root          = '',
-  $environments                      = {},
-  $portstrees                        = {},
+  String                       $zpool                 = 'tank',
+  Stdlib::Absolutepath         $zrootfs               = '/poudriere',
+  String                       $freebsd_host          = 'http://ftp6.us.freebsd.org/',
+  Stdlib::Absolutepath         $resolv_conf           = '/etc/resolv.conf',
+  Boolean                      $ccache_enable         = false,
+  Stdlib::Absolutepath         $ccache_dir            = '/var/cache/ccache',
+  Stdlib::Absolutepath         $poudriere_base        = '/usr/local/poudriere',
+  String                       $poudriere_data        = '${BASEFS}/data',
+  Enum['yes', 'no']            $use_portlint          = 'no',
+  Optional[String]             $mfssize               = undef,
+  Enum['yes', 'no']            $tmpfs                 = 'yes',
+  Stdlib::Absolutepath         $distfiles_cache       = '/usr/ports/distfiles',
+  Optional[String]             $csup_host             = undef,
+  Optional[String]             $svn_host              = undef,
+  Enum['yes', 'no', 'verbose'] $check_changed_options = 'verbose',
+  Enum['yes', 'no']            $check_changed_deps    = 'yes',
+  String                       $pkg_repo_signing_key  = '',
+  Integer[1]                   $parallel_jobs         = $facts['processors']['count'],
+  String                       $save_wrkdir           = '',
+  String                       $wrkdir_archive_format = '',
+  String                       $nolinux               = '',
+  String                       $no_package_building   = '',
+  String                       $no_restricted         = '',
+  String                       $allow_make_jobs       = '',
+  String                       $url_base              = '',
+  String                       $max_execution_time    = '',
+  String                       $nohang_time           = '',
+  String                       $http_proxy            = '',
+  String                       $ftp_proxy             = '',
+  String                       $build_as_non_root     = '',
+  Hash                         $environments          = {},
+  Hash                         $portstrees            = {},
```

Comment:
I'm not sure if I like the whole indentation. It adds a lot of noise to the PR and future changes to the datatypes or new parameters might require more whitespace on all the other params.

Reply:
I had this in 2 commits, one for the actual changes and another to align fields properly.
I fsck'ed up this when integrating @igalic suggestions, so I will re-do this cleanly with your suggestions.